third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-successful-run-expected.txt

```diff
@@ -195,6 +199,9 @@ Computing artifact: UserTimings
 Auditing: Avoid chaining critical requests
 Computing artifact: CriticalRequestChains
 Auditing: Avoid multiple page redirects
+Computing artifact: LanternInteractive
+Computing artifact: LanternFirstMeaningfulPaint
```

Comment:
These are getting computed twice now. I think it's because `gatherMode` is being passed in to these computed artifacts now. Can it be avoided?

Reply:
Ah good catch, yeah the issue is one too many properties in the internal TBT computation version 👍

Reply:
well that turned into a bit more of a rabbit hole then I expected 😅  but cleaned up a few of our existing issues while I was at it